KubeArmor/eventAuditor/entryPointHandler.go

```diff
@@ -132,30 +133,44 @@ func (ea *EventAuditor) DestroyEntryPointPrograms(bman *KABPFManager) error {
 func (ea *EventAuditor) EnableEntryPoint(probe string) {
 	var eventMapElem EventElement
 
+	probe = strings.Replace(probe, "sys_", "", -1)
```

Comment:
if strings.HasPrefix(probe, "sys_") {
   ...
}
we need to make a room for other kprobes too.

Reply:
Okay, will change it